TSS.NET/TSS.Net/CryptoAsym.cs

```diff
@@ -351,32 +250,17 @@ public ISignatureUnion SignData(byte[] data, TpmAlgId sigHash)
                         {
                             sigHash = (rsaParams.scheme as SigSchemeRsassa).hashAlg;
                         }
-                        byte[] digest = CryptoLib.HashData(sigHash, data);
-#if TSS_USE_BCRYPT
-                        byte[] sig = Key.SignHash(digest, BcryptScheme.Rsassa, sigHash);
-#else
-                        byte[] sig = RsaProvider.SignData(data, CryptoLib.GetHashName(sigHash));
-#endif
+                        byte[] sig = RsaProvider.SignData(data, CryptoLib.GetHashAlgorithmName(sigHash), RSASignaturePadding.Pkcs1);
                         return new SignatureRsassa(sigHash, sig);
                     }
                     case TpmAlgId.Rsapss:
                     {
-#if true
-                        Globs.Throw<ArgumentException>("SignData(): PSS scheme is not supported");
-                        return null;
-#else
                         if (sigHash == TpmAlgId.Null)
                         {
-                            sigHash = (rsaParams.scheme as SigSchemeRsapss).hashAlg;
+                            sigHash = (rsaParams.scheme as SigSchemeRsassa).hashAlg;

```

Comment:
This should stay as `SigSchemeRsapss`, right?

Reply:
Great catch, thank you. Fixed.